websocket_server/appinfo/info.xml

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0"?>
+<!--
+  - SPDX-FileCopyrightText: 2024 Nextcloud GmbH and Nextcloud contributors
+  - SPDX-License-Identifier: AGPL-3.0-or-later
+-->
+<info>
+    <id>whiteboard_websocket</id>
```

Comment:
@oleksandr-nc Is it possible to have an ext app shipped as part of an existing regular PHP app? So if the ext app ecosystem is installed it will just run the docker container in addition to the existing logic?

Reply:
At the moment there is no such functionality, but if there is a request for implementing this, and Daphne approve this, Andrey or I can add this feature.

Reply:
I guess https://github.com/nextcloud/app_api is the right place?